Assets/Tests/PlayMode/DetectionTests.cs

```diff
@@ -15,10 +15,60 @@ public void SetUp()
             SceneManager.LoadScene("MainScene");
         }
 
+         /// <summary>
+        /// Tests if the tower can detect an enemy in Range
+        /// </summary>
+        /// <returns></returns>
         [UnityTest]
-        public IEnumerator TowersDetecEnemies()
+        public IEnumerator TowersDetectsEnemies()
         {
+            GameObject tower = GameObject.FindWithTag("Tower");
+            GameObject enemy = GameObject.FindWithTag("Enemy");
+            float distance = tower.GetComponent<TowerBattle>().Range;
+            //places the enmey within tower's range
+            enemy.transform.position = tower.transform.position + new Vector3(distance / 2, 0, distance / 2);
+            yield return new WaitForSeconds(0.5f);
+            Assert.AreEqual(tower.GetComponent<Detection>().isOccupied, true);
```

Comment:
Next time just flip these arguments because the syntax for `AreEqual` is expected, then actual